docs/how-to/how-to-run-dind-with-kata.md

```diff
@@ -0,0 +1,122 @@
+# How to run Docker in Docker with Kata Containers
```

Comment:
- Please add a link to this document from `docs/how-to/README.md` or the CI will fail this PR as the doc is "unreferenced".
- I would consider adding a "Note" (see https://github.com/kata-containers/kata-containers/blob/main/docs/Documentation-Requirements.md#notes) somewhere near the top of this document explaining that you can't actually run Kata 2.x with Docker to avoid misleading the reader.

Reply:
- [ ] Add note regarding the inability to run Kata 2.x with docker

Reply:
@jodh-intel do you think maybe I should rename this nested containerization or something? The whole "Docker in Docker" phrasing itself might be a little misleading in and of itself

Reply:
I've gone with `how-to-run-docker-with-kata.md` as the title and will alter references to DinD